rcl/test/rcl/test_subscription.cpp

```diff
@@ -213,7 +213,7 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
     ret = rcl_publish(&publisher, &msg);
     // TODO(wjwwood): re-enable this fini when ownership of the string is resolved.
     //                currently with Connext we will spuriously get a double free here.
-    // std_msgs__msg__String__fini(&msg);
+    std_msgs__msg__String__fini(&msg);
```

Comment:
Yeah, I believe I do this in this pull request: https://github.com/ros2/rmw_connext/pull/164/files#diff-d3deaebdd6625290a633bebc6ead9d88R232